Survey/Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -50,14 +47,26 @@ struct LoginView: View {
                     )
                     .accessibilityIdentifier("loginButtonIdentifier")
                 }
-                .alert(isPresented: $isLoginFailed) { () -> Alert in
+                .alert(isPresented: $viewModel.isLoginFailed) { () -> Alert in
                     Alert(
                         title: Text(Localize.login_failed_title()),
-                        message: Text(Localize.login_failed_description())
+                        message: Text(viewModel.errorMessage)
                     )
                 }
                 .offset(y: editingMode ? -140 : -60)
```

Comment:
```suggestion
                .offset(y: editingMode ? -140.0 : -60.0)
```

Reply:
done in this [commit](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/44/commits/f8b1f24fbec87c8dd42a17c31afcdba49baa20f7)